### ISSUE TITLE
Please tag latest version

### ISSUE THREAD

**Issue by emyr666** [facebook/folly#143]
Hi,
There are a few "bumped version" commit logs but there doesn't seem to be any tags > v0.24.0.
Some facebook projects which use folly (fbthrift & proxygen) have their build systems pulling in this older version and they suffer from the at_fork unavailable issue which has now been fixed in the folly codebase. It would be great if the fb developers of these 3 projects (i.e. the above 2 and folly could work together so that we can get versions of fbthrift and proxygen out that have the fix.
Basically this is a call to get this https://github.com/facebook/folly/pull/125 filtered to the upstream projects as soon as possible.
Many thanks,
Emyr

**Comment by emyr666** [comment]
Tags up to v0.27.0 have now appeared. Many thanks. Next step...I will try to get fbthrift and proxygen to pull in later version sof folly....

**Comment by andreazevedo** [comment]
Tags up to v0.31.0 are up.
